x/multi-staking/module.go

```diff
@@ -117,29 +122,18 @@ func (am AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {
 	multistakingkeeper.RegisterInvariants(ir, am.keeper)
 }
 
-// Deprecated: Route returns the message routing key for the staking module.
-func (am AppModule) Route() sdk.Route {
-	return sdk.Route{}
-}
-
 // QuerierRoute returns the staking module's querier route name.
 func (AppModule) QuerierRoute() string {
 	return multistakingtypes.QuerierRoute
 }
 
-// LegacyQuerierHandler returns the staking module sdk.Querier.
-// TODO: add legacy querier
-func (am AppModule) LegacyQuerierHandler(legacyQuerierCdc *codec.LegacyAmino) sdk.Querier {
-	return nil
-}
-
 // RegisterServices registers a GRPC query service to respond to the
 // module-specific GRPC queries.
 func (am AppModule) RegisterServices(cfg module.Configurator) {
 	stakingtypes.RegisterMsgServer(cfg.MsgServer(), multistakingkeeper.NewMsgServerImpl(am.keeper))
 	multistakingtypes.RegisterQueryServer(cfg.QueryServer(), multistakingkeeper.NewQueryServerImpl(am.keeper))
 
-	querier := stakingkeeper.Querier{Keeper: am.sk}
+	querier := stakingkeeper.Querier{Keeper: &am.sk}
```

Comment:
We not register staking module so in this situation, staking keeper subspace param will not migrate because the migration is register [here](https://github.com/cosmos/cosmos-sdk/blob/b68da64d8e38caa0438dbf3b065d864d91e03c4b/x/staking/module.go#L154-L163). So we need to add migration from staking module to multi-staking